src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/Attributes.cs

```diff
@@ -1,129 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-
-namespace System.Runtime.InteropServices
-{
-    [AttributeUsage(AttributeTargets.Assembly | AttributeTargets.Class | AttributeTargets.Interface, Inherited = false)]
-    public sealed class AutomationProxyAttribute : Attribute
```

Comment:
I think these attributes should stay here - they should be removed from CoreLib instead.